geometry_msgs/Pose Tnow
geometry_msgs/Pose Tlast_fuse
geometry_msgs/Pose Todom
ndt_map/NDTMapMsg map #the map
uint32 ctr
